README.md

```diff
@@ -44,17 +44,17 @@ Nim is an Open Source programming language in which many Open Source projects ar
 
 **E-mail**: *we can skip this in this repo*
 
-**Submitter’s affinity to the topic of the devroom**: Core Developer
+**Submitter’s affinity to the topic of the devroom**: core developer, author of “Nim in Action” book
 
 ### Secondary contact
 
-**First name**
+**First name** Pietro
 
-**Last name**
+**Last name** Peterlongo
 
 **E-mail**: *we can skip this in this repo*
 
-**Additional pentabarf ids:** pietroppeter, federico3
+**Additional pentabarf ids:** pietroppeter, federico3, moigagoo, xigoi
```

Comment:
did you create your pentabarf account using fosdem system (https://penta.fosdem.org/user/new_account)?
do you know if @xigoi has created a pentabarf account with same user of GitHub or forum?

Reply:
No problem!

> If you can confirm you have created a pentabarf account we should add it to the list

How do I get it? Googling doesn't help much.

Reply:
through the link I posted in the comment 😉: https://penta.fosdem.org/user/new_account

Reply:
> through the link I posted in the comment 😉: https://penta.fosdem.org/user/new_account

Sorry, I missed it.

> did you create your pentabarf account using fosdem system (https://penta.fosdem.org/user/new_account)?

I've created the account. The account name is moigagoo.

Reply:
I've created my account now.